### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.{js,html}]
+indent_style = space
+indent_size = 4
```
Comment (mr-mig) [inline .editorconfig]: I'd better use 2 spaces for js

Comment (vergilius) [inline .editorconfig]: nah, I'd go with 4 ;)
@mr-mig why would you like it to be 2?

**Comment by vergilius** [comment]
:+1:

File: .editorconfig (hunk 1 above)
Comment (mr-mig) [inline .editorconfig]: I used to 2 spaces.
Do you want javascript be closer to java? :trollface:

Comment (kdzwinel) [inline .editorconfig]: not sure why we are even discussing it, we use 4 spaces in web-client and symfony and IMO we should stick to it, it should be possible to adjust visual space in IDE

Comment (mr-mig) [inline .editorconfig]: https://github.com/facebook/react/blob/master/.editorconfig

**Comment by kdzwinel** [comment]
:+1:

**Pull request merged**